src/main/java/com/o19s/es/ltr/query/StoredLtrQueryBuilder.java

```diff
@@ -106,6 +109,7 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeOptionalString(modelName);
         out.writeOptionalString(featureSetName);
         out.writeMap(params);
+        out.writeMap(activeFeatures);
```

Comment:
gating this on the elasticsearch version doesn't seem right, as the important bit is the version of the plugin.  It seems plausible the code gets backported to another es version. Maybe YAGNI though i'm not sure.  Unfortunately i also have no suggestions for how to gate this properly on our plugin versions. The only thing i can think of would have required us storing a version header on the actual serialized objects, and that would still be iffy.

Reply:
true, sadly I see no other options for now. This kind of version check basically prevents backports `Version.V_6_2_3` will trigger compilation errors. 
We might consider wrapping all the streams with our own Stream[In/Out]put when moving to 7 so that we can store the plugin version as you suggest allowing perhaps to backport more easily.

As for this specific PR we can perhaps do what is wanted without changing the binary format.
If we agree on a *magic name* we could store this feature filter inside a param of the sltr query itself. A name like `__ltr_active_features__`.
The query would look like:
```json
"sltr": {
  "params": {
         "query_string": "user query",
         "random_param": { },
         "__ltr_active_features__": [ "feat1", "feat2" ],
  },
  "model": "foobar"
}
```

I'm perfectly fine with this solution assuming it's well documented.

Reply:
fwiw this PR embarks the version bump to 6.2.3 so this is a guarantee that this plugin is not deployed anywhere on es 6.2.3 without this code.

Reply:
oops I see 6.2.3 in http://es-learn-to-rank.labs.o19s.com/ this means a release have been made without a commit on the master branch :(
In other words this version check will prevent users using this plugin on 6.2.3 to do a rolling upgrade.
I might suggest to use the magic param solution or bump to another elastic version.

Reply:
Updated to use elasticsearch version 6.2.4 or after.